tasks/ast_tools/src/generators/raw_transfer.rs

```diff
@@ -25,10 +25,9 @@ use crate::{
 use super::define_generator;
 
 // Offsets of `Vec`'s fields.
-// These are based on observation, and are not stable.
-// They will be fully reliable once we implement our own `Vec` type and make it `#[repr(C)]`.
+// The `Vec` has marked `#[repr(C)]` layout, so the offsets are fixed.
 const VEC_PTR_FIELD_OFFSET: usize = 0;
-const VEC_LEN_FIELD_OFFSET: usize = 24;
+const VEC_LEN_FIELD_OFFSET: usize = 8;
```

Comment:
This needs to go back to 24. Test NAPI CI task has not failed but it's still running. Probably it's in an infinite loop.

Reply:
I've fixed this.